### DISCUSSION TITLE
where to define the conifgurations ?

### DISCUSSION THREAD

**Discussion by lamaclas** [Shopify/prettier-plugin-liquid#170] [Q&A] [answered]
Where can i change the default configuration values? 
for example i want to useTabs to be true

**Comment by charlespwd** [comment, marked as answer]
By using a [configuration file](https://prettier.io/docs/en/configuration.html). We integrate with the prettier one.

You can also set [overrides](https://prettier.io/docs/en/configuration.html#configuration-overrides) for Liquid if some of the rules are contradictory.
